masterlist.yaml

```diff
@@ -13931,9 +13931,16 @@ plugins:
       - 'FCO - Follower Commentary Overhaul.esp'
       - 'Relationship Dialogue Overhaul.esp'
     tag: [ Actors.Voice ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x82652DA7
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 39
     clean:
       - crc: 0xFFFB3713
         util: 'SSEEdit v4.0.3'
+      - crc: 0x3B8BB25A
+        util: 'SSEEdit v4.0.4'
```

Comment:
We don't include the clean checksum for dirty plugins as it can be volatile. The old cleaning data can be removed as well:

```suggestion
```

Reply:
Updated